validator.py

```diff
@@ -0,0 +1,16 @@
+"""Validation things"""
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
Same comment as above I think some of these imports are unnecessary here. Should check the other files too.